lib/cli.js

```diff
@@ -574,6 +574,42 @@ program
   })
 
 program
+  .command('auth-openid')
+  .description('start openid auth client')
+  .option('-p, --port <port>'
+    , 'port (or $PORT)'
+    , Number
+    , process.env.PORT || 7120)
+  .option('-s, --secret <secret>'
+    , 'secret (or $SECRET)'
+    , String
+    , process.env.SECRET)
+  .option('-a, --app-url <url>'
+    , 'URL to app'
+    , String)
+  .option('--identifier <identifier>'
+    , 'identifier'
```

Comment:
It should be easy to understand. Some thing like this `OpenID Identifier URL (or $OPENID_IDENTIFIER_URL)`